src/pykeen/models/unimodal/tucker.py

```diff
@@ -86,35 +84,29 @@ class TuckER(EntityRelationEmbeddingModel):
 
     def __init__(
         self,
-        triples_factory: CoreTriplesFactory,
+        *,
         embedding_dim: int = 200,
         relation_dim: Optional[int] = None,
-        loss: Optional[Loss] = None,
-        preferred_device: DeviceHint = None,
-        random_seed: Optional[int] = None,
         dropout_0: float = 0.3,
         dropout_1: float = 0.4,
         dropout_2: float = 0.5,
-        regularizer: Optional[Regularizer] = None,
         apply_batch_normalization: bool = True,
         entity_initializer: Hint[Initializer] = xavier_normal_,
         relation_initializer: Hint[Initializer] = xavier_normal_,
+        **kwargs,
     ) -> None:
         """Initialize the model.
 
         The dropout values correspond to the following dropouts in the model's score function:
 
+        .. math::
+
             DO_2(BN(DO_0(BN(h)) x_1 DO_1(W x_2 r))) x_3 t
```

Comment:
If we use latex, `x_1` should be `\times_1`.

Reply:
Ahh funny. The notation for this is so goofy - the operation itself is indexed!

Reply:
It is used to denote the product along the 1st mode of the tensor. (`x_2` and `x_3` have the same meaning in this formula).